arch/xtensa/src/esp32s2/hardware/esp32s2_rtccntl.h

```diff
@@ -4418,29 +5030,44 @@
 #define RTC_CNTL_FORCE_DOWNLOAD_BOOT_S  0
 
 /* RTC_CNTL_SLP_WAKEUP_CAUSE_REG register
- * sleep wakeup cause state register
+ * Stores the sleep-to-wakeup cause.
  */
 
 #define RTC_CNTL_SLP_WAKEUP_CAUSE_REG (DR_REG_RTCCNTL_BASE + 0x12c)
 
 /* RTC_CNTL_WAKEUP_CAUSE : RO; bitpos: [16:0]; default: 0;
- * sleep wakeup cause
+ * Stores the wakeup cause.
  */
 
-#define RTC_CNTL_WAKEUP_CAUSE    0x0001FFFF
+#define RTC_CNTL_WAKEUP_CAUSE    0x0001ffff
```

Comment:
It's a bit confusing why part of the files use lower case and other part an upper case